docs/tutorials/quickstart/quick-start-compiler.md

```diff
@@ -9,23 +9,23 @@
 
 1. Double click on the `MBED.HTML` file. This adds your Mbed board to the Online Compiler as a compilation target.
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/add_to_compiler2.png)</span>
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/add_to_compiler.png)</span>
 
 ### Importing the code
 
 Click the button below to automatically import the example into the Online Compiler.
 
-[![View Example](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-quick-start-blinky)](https://github.com/ARMmbed/mbed-os-quick-start-blinky/blob/master/main.cpp)
+[![View Example](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-blinky)](https://github.com/ARMmbed/mbed-os-example-blinky/blob/master/main.cpp)
```

Comment:
Can just got back to the old school go the teams page to import image here as well now since blinky is already there once the PR is merged and gets migrated over.